app/scripts/controllers/network/createJsonRpcClient.js

```diff
@@ -12,21 +12,25 @@ import { SECOND } from '../../../../shared/constants/time';
 
 const inTest = process.env.IN_TEST;
 const blockTrackerOpts = inTest ? { pollingInterval: SECOND } : {};
-const getTestMiddlewares = () => {
-  return inTest ? [createEstimateGasDelayTestMiddleware()] : [];
-};
 
-export default function createJsonRpcClient({ rpcUrl, chainId }) {
+export default function createJsonRpcClient({ rpcUrl, chainId, networkId }) {
```

Comment:
What do you think about making API/behavior changes to `createJsonRpcClient` in a different PR? That way it might be easier to follow why we are doing this.